test/unit/spec/v1/IDPDiscovery_spec.js

```diff
@@ -1192,27 +1136,6 @@ Expect.describe('IDPDiscovery', function() {
           );
         });
     });
-    itp('removes anti-phishing message if help link is clicked', function() {
-      return setup({
-        baseUrl: 'http://foo<i>xss</i>bar.com?bar=<i>xss</i>',
-        features: { securityImage: true, selfServiceUnlock: true },
-      })
-        .then(function(test) {
-          test.setNextResponse(resSecurityImageFail);
-          test.form.setUsername('testuser@clouditude.net');
-          return waitForBeaconChange(test);
-        })
-        .then(function(test) {
-          // Tooltip exists
-          expect(test.form.isSecurityImageTooltipDestroyed()).toBe(false);
-          spyOn(test.router, 'navigate');
-          test.form.helpFooter().click();
-          test.form.unlockLink().click();
-          expect(test.router.navigate).toHaveBeenCalledWith('signin/unlock', { trigger: true });
-          // Verify tooltip is gone
-          expect(test.form.isSecurityImageTooltipDestroyed()).toBe(true);
-        });
-    });
```

Comment:
Are the deleted tests in this file moved someplace else or are they removed entirely?

Reply:
Yes sir, moved into the PrimaryAuthForm_spec.js testcafe spec: https://github.com/okta/okta-signin-widget/pull/3418/files#diff-7fc65ff7a9deb6e52de14ecbe846e256708dd5e763b6a70031bd7cbe0f57a5b7R120-R226